caleb-lab/routes/person-routes.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const createError = require('http-errors');
+const personCtrl = require('../controllers/person-controller.js');
+
+
+module.exports = function(router){
+
+  //all
+  router.get('/api/person/', (req, res) => {
+
+    personCtrl.fetchPeople()
+    .then(person => {
+      console.log(person);
+      res.json(person);
+    })
+    .catch(err => res.status(404).send(err.message));
+  });
+
+  //get
+  router.get('/api/person/:id', (req, res) => {
+    if(!req.params.id) return res.status(400).send(createError('you done fucked up, boy.'));
+    personCtrl.fetchPerson(req.params.id)
+    .then(person => {
+      console.log(person);
+      res.json(person);
+    })
+    .catch(err => res.status(404).send(err.message));
+  });
+
+
+  // post new person
+  router.post('/api/person/:id', (req, res) => {
```

Comment:
**TODO FOR LOST POINTS**

Take out the ``:id`` part here at Line 33. You don't get an ID generated until you post the new Person, so this endpoint shouldn't take a specific ID. 

When I took out that ``:id`` your post worked successfully, so good work on that.